lib/gem_of.rb

```diff
@@ -53,11 +48,20 @@ def initialize
         eval(File.read(user_gemfile), binding)
       end
       HEREDOC
-      # rubycritic has trouble in older rubys because of transitive deps
-      unless Gem::Version.new(RUBY_VERSION).between?(Gem::Version.new("2.0.0"),
-                                                     Gem::Version.new("2.1.5"))
+      # we need nokogiri for old versions of ruby (and beaker before 4)
```

Comment:
Can this use the the following?
```
      if Gem::Version.new(RUBY_VERSION).between?(Gem::Version.new("2.0.0"),
                                                 Gem::Version.new("2.2.4"))
...
```

Reply:
heh.  good catch.  running fast and loose over here with old logic.  fixing...